ntoskrnl/mm/i386/page.c

```diff
@@ -269,58 +279,53 @@ MmDeleteVirtualMapping(PEPROCESS Process, PVOID Address,
         ASSERT(Process = PsGetCurrentProcess());
```

Comment:
This looks like a bug, since there is an assignment (instead of an equality check) done within the ASSERT, meaning also, that this problem would happen only on DBG builds, not release ones.

Reply:
Uh. IndeedN

Reply:
And same typo at old line 634.